src/main/viewManager.js

```diff
@@ -184,4 +186,57 @@ export class ViewManager {
       hideView();
     };
   }
+
+  setFinderBounds = () => {
+    if (this.finder) {
+      const currentWindow = this.getCurrentView().window;
+      const boundaries = currentWindow.getBounds();
+      this.finder.setBounds({
+        x: boundaries.width - FINDER_WIDTH - (process.platform === 'darwin' ? 20 : 200),
+        y: 0,
+        width: FINDER_WIDTH,
+        height: FINDER_HEIGHT,
+      });
+    }
+  }
+
+  focusFinder = () => {
+    if (this.finder) {
+      this.finder.webContents.focus();
+    }
+  }
+
+  hideFinder = () => {
+    if (this.finder) {
+      const currentWindow = this.getCurrentView().window;
+      currentWindow.removeBrowserView(this.finder);
+      this.finder.destroy();
+      this.finder = null;
+    }
+  }
+
+  foundInPage = (result) => {
+    if (this.finder) {
+      this.finder.webContents.send(FOUND_IN_PAGE, result);
+    }
+  }
+
+  showFinder = () => {
+    // don't open another finder if one is already open
+    if (this.finder) {
+      return;
+    }
```

Comment:
if the user hit the shorcut a second time, should we send the focus to the finder? and ideally it might select the whole text within the textbox?

Reply:
That sounds like a good idea, will implement.